plugins/wasp/general-wasp-knowledge.md

```diff
@@ -0,0 +1,126 @@
+# Wasp App
+
+This project uses TypeScript and Wasp, a batteries-included framework for building full-stack web apps with React, Node.js, and Prisma.
+
+## Documentation
+
+Your training data about Wasp may be outdated. You MUST fetch and verify against the current documentation before implementing Wasp features or answering Wasp questions:
+- https://wasp.sh/llms.txt (latest)
+- https://wasp.sh/llms-{version}.txt (versioned)
+
+## Workflows
+
+### Development Server
+
+Before starting any development task on this Wasp project (coding, debugging, adding features), you MUST check if the dev server is running:
+- Check for active background tasks (use `/tasks` command)
+- Or check if ports 3000 (client) and 3001 (server) are in use
+
+If not running, invoke the `start-dev-server` skill to start the server as a background task or externally.
+
+### Implementation Checklist
+
+1. Fetch https://wasp.sh/llms.txt to verify current syntax (see "Documentation" above)
+2. Check if the feature exists in the current codebase or starter template for reference patterns
+3. Identify which files need changes: wasp config file (`main.wasp` or `main.wasp.ts`), `schema.prisma`, `src/`
+4. Plan the order: schema → migrate → wasp config file → TypeScript
```

Comment:
What if there are no schema changes, will this still push it to force them a bit, I hope not?

Btw could be Javascrit potentially and not Typescript. I woudlnt' recommend it, especially with AI, but it is possible.